index.html

```diff
@@ -50,108 +50,44 @@
 </script> -->
 
 <div class="row">
-<div class="col-md-12">
-<p>
-QuTiP is open-source software for simulating the dynamics of open quantum systems.  The QuTiP library depends on the excellent <a href='https://www.numpy.org'>Numpy</a>, <a href='https://www.scipy.org'>Scipy</a>, and <a href='https://cython.org'>Cython</a> numerical packages. In addition, graphical output is provided by <a href='https://matplotlib.org'>Matplotlib</a>.  QuTiP aims to provide user-friendly and efficient numerical simulations of a wide variety of Hamiltonians, including those with arbitrary time-dependence, commonly found in a wide range of physics applications such as quantum optics, trapped ions, superconducting circuits, and quantum nanomechanical resonators. QuTiP is freely available for use and/or modification on all major platforms such as Linux, Mac OSX, and Windows<sup>*</sup>. Being free of any licensing fees, QuTiP is ideal for exploring quantum mechanics and dynamics in the classroom.
-</p>
-<p style='font-size:10px'><sup>*</sup>QuTiP is developed on Unix platforms only, and some features may not be available under Windows.</p>
-</div>
-</div>
-
-<div class="row" style='margin-top:20px'>
-<div class="col-md-6">
-<img src="images/corp_users.png">
-</div>
-<div class="col-md-5">
-<p>
-From the classroom to the corporate office, QuTiP is used around the world to advance research in quantum optics, quantum information, quantum open systems, optomechanics, and many related fields.  QuTiP is in use at nearly every single research university around the globe, government-funded research labs, and is relied upon by every major corporation focused on developing a quantum computer.
-</p>
-</div>
+    <div class="col-md-12">
+        <p>
+            QuTiP is open-source software for simulating the dynamics of open quantum systems.  The QuTiP library depends on the excellent <a href='https://www.numpy.org'>Numpy</a>, <a href='https://www.scipy.org'>Scipy</a>, and <a href='https://cython.org'>Cython</a> numerical packages. In addition, graphical output is provided by <a href='https://matplotlib.org'>Matplotlib</a>.  QuTiP aims to provide user-friendly and efficient numerical simulations of a wide variety of Hamiltonians, including those with arbitrary time-dependence, commonly found in a wide range of physics applications such as quantum optics, trapped ions, superconducting circuits, and quantum nanomechanical resonators. QuTiP is freely available for use and/or modification on all major platforms such as Linux, Mac OSX, and Windows<sup>*</sup>. Being free of any licensing fees, QuTiP is ideal for exploring quantum mechanics and dynamics in the classroom.
+        </p>
+        <p style='font-size:10px'><sup>*</sup>QuTiP is developed on Unix platforms only, and some features may not be available under Windows.</p>
```

Comment:
Is this really true? Where did it come from? We test on Windows these days and officially release packages for Windows.

Reply:
It's on the current main homepage actually. But I can just remove it.